### PR TITLE
Add lawyers controller to sign up as a lawyer

### PR DESCRIPTION
**Pull request by bouk**
@kwestein 

This adds a page where you can sign up as a lawyer. For now it just generates a password

I also added some displaying of the flashes that rails creates

### CODE REVIEW

**Comment by kwestein** [comment]
@OttawaU-RSSP/team

File: app/controllers/lawyers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class LawyersController < ApplicationController
+  def new
+    @lawyer = Lawyer.new
+  end
+
+  def create
+    @lawyer = Lawyer.new(lawyer_params)
```
Comment (kwestein) [inline app/controllers/lawyers_controller.rb:7]: Can this just be an instance variable?

**Comment by kwestein** [comment]
:+1:

File: app/controllers/lawyers_controller.rb (hunk 1 above)
Comment (bouk) [inline app/controllers/lawyers_controller.rb:7]: If the validation fails I render new, which uses `@lawyer` to populate any fields that were already filled in

**Pull request merged**